e2e/cypress/integration/markdown/markdown_text_spec.js

```diff
@@ -65,7 +65,7 @@ describe('Markdown message', () => {
 </blockquote>`;
 
         // #  Post markdown message
-        cy.postMessageFromFile('markdown/markdown_block_quotes_2.md');
+        cy.postMessageFromFile('markdown/markdown_block_quotes_2.md').wait(TIMEOUTS.SMALL);
```

Comment:
Explicitly add wait after posting a message.  Previously, we're waiting more [before posting it](https://github.com/mattermost/mattermost-webapp/pull/3631/files#diff-ecaf15e463c339446fc523beb333c97aR171).